src/BenchmarksServer/Startup.cs

```diff
@@ -1105,7 +1118,7 @@ async Task StopJobAsync()
                                     }
                                     else if (OperatingSystem == OperatingSystem.Linux)
                                     {
-                                        // TODO: Stop perfcollect
+                                        await StopPerfcollectAsync(perfCollectProcess);
```

Comment:
@sebastienros this is a quite an important change. So far we were stopping perfcollect later, when client was asking for trace file.